### PR TITLE
Small adjustments and NTP

### PR DESCRIPTION
**Pull request by goetzc**
* Update copyright year to 2017
* Use systemd-timesyncd as NTP client and to set the time zone
  * This removes geerlingguy.ntp dependency
* The "become_method: sudo" is the default privilege escalation method
* Add validate for lineinfile module
* Fix small typos and add more descriptive names for modules

### CODE REVIEW

**Comment by goetzc** [comment]
@lpaulmp please review.

File: tasks/bootstrap-tasks.yml [hunk 1]
```diff
@@ -35,6 +35,12 @@
   tags:
     - set-hostname
 
+- name: Set time zone
+  command: timedatectl set-timezone {{ ntp_timezone }}
+
+- name: Enable NTP
+  command: timedatectl set-ntp true
```
Comment (lpaulmp) [inline tasks/bootstrap-tasks.yml]: @goetzc  Which is the remote host that this server is synchronizing?

**Comment by goetzc** [comment]
The Ansible version is 2.2.1.0 (which is the latest stable).

File: tasks/bootstrap-tasks.yml (hunk 1 above)
Comment (goetzc) [inline tasks/bootstrap-tasks.yml]: It uses the default Debian pool from ntp.org which is defined at systemd compile-time.[1] In addition, the user can configure custom servers in `/etc/systemd/timesyncd.conf`.

```
[Time]
NTP=0.debian.pool.ntp.org 1.debian.pool.ntp.org 2.debian.pool.ntp.org 3.debian.pool.ntp.org
```
[1] https://sources.debian.net/src/systemd/215-17%2Bdeb8u6/debian/rules/#L28

**Pull request merged**